src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -199,11 +198,23 @@ class JesBackend extends Backend with LazyLogging {
       case (fileTag: String, location: WdlFile) =>
         logger.info(s"${makeTag(backendCall)}: $fileTag -> ${location.valueString}")
         Seq(mkInput(fileTag, location))
-      case (fileTag: String, location: WdlArray) => location.value.collect { case f: WdlFile => mkInput(fileTag, f) }
+      case (fileTag: String, localPathWdlArray: WdlArray) =>
+        backendCall.lookupFunction(fileTag) match {
+          case WdlArray(wdlType: WdlArrayType, remotePathArray: Seq[WdlValue]) =>
+            jesInputsFromArray(fileTag, remotePathArray, localPathWdlArray.value)
+        }
       case (fileTag: String, location: WdlMap) => location.value flatMap { case (k, v) => Seq(k, v) } collect { case f: WdlFile => mkInput(fileTag, f) }
     } flatten
   }
 
+  private def jesInputsFromArray(jesNamePrefix: String, remotePathArray: Seq[WdlValue], localPathArray: Seq[WdlValue]): Iterable[JesInput] = {
```

Comment:
There is zero need to test a private function. Whatever calls it would be tested and we don't care how the actual work gets done